app/src/main/java/org/mozilla/fenix/settings/search/SearchEngineFragment.kt

```diff
@@ -60,13 +60,19 @@ class SearchEngineFragment : PreferenceFragmentCompat() {
         val searchEngineListPreference =
             findPreference<SearchEngineListPreference>(getPreferenceKey(R.string.pref_key_search_engine_list))
 
+        val showVoiceSearchPreference =
```

Comment:
Do we also want to hide this pref behind the feature flag?

Reply:
Good call!